<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginCreate</class>
 <widget class="QWidget" name="PluginCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PluginCreate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>插件名称:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pluginName_Let"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>插件版本:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="version_Let"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>插件路径:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="setPath_Let"/>
     </item>
     <item>
      <widget class="QToolButton" name="setPath_Btn">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_Btn">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_Btn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
